Gemfile

```diff
@@ -110,7 +106,6 @@ gem 'bootsnap', require: false
 # Technically they don't need to be in this Gemfile at all, but we are pinning them to
 # specific versions for compatibility reasons.
 gem "nokogiri", ">= 1.10.4"
-gem "image_processing"
```

Comment:
supposed to be required for compatibility reasons, but it isn't a dependency of any other gem.

Reply:
also if it was a dependency, then it still wouldn't do anything because the version isn't pinned.